src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -158,7 +205,7 @@ CAC.Pages.Home = (function ($, BikeModeOptions,  MapControl, Templates, UserPref
         event.preventDefault();
 
         // hide existing destinations list and show loading spinner
-        $(options.selectors.destinationsContainer).addClass('hidden');
+        $(options.selectors.placeList).addClass('hidden');
```

Comment:
There is a bunch of old code that has been retained. I've been holding off on purging things generally until some more of the new functionality is stable, as some of it we do want or will break something else if removed immediately.